src/paymentservice/package.json

```diff
@@ -12,15 +12,15 @@
   "license": "ISC",
   "dependencies": {
     "@google-cloud/debug-agent": "5.1.3",
-    "@google-cloud/profiler": "4.1.1",
+    "@google-cloud/profiler": "4.1.5",
```

Comment:
This upgraded needed because of [this error](https://github.com/GoogleCloudPlatform/microservices-demo/runs/4299781584?check_suite_focus=true) in the CI pipeline

The [v3.0.0./node-v93-linux-x64-musl.tar.gz] artifact of `pprof-nodejs` is not anymore found in the [artifacts bucket](https://storage.googleapis.com/cloud-profiler)

Reply:
Thanks for figuring this out!